### PR TITLE
Use targetURI for cloudEvent resource details

### LINKED ISSUE #383: Display targetURL in list for cloudEvent resource
# Expected Behavior
When listing resources, key details are displayed in the last column, but not for cloudEvent pipeline resources.

# Actual Behavior

```
$ tkn resources list

gsutil-image                  image        url: cmd/gsutil
ko-image                      image        url: ko-ci
kubeconfigwriter-image        image        url: cmd/kubeconfigwriter
nop-image                     image        url: cmd/nop
pipeline-git-v0-7-0           git          url: https://github.com/tektoncd/pipeline
plumbing-git-master           git          url: https://github.com/afrittoli/plumbing
post-release-trigger          cloudEvent   ---
pull-request-init-image       image        url: cmd/pullrequest-init
skaffold-git                  git          url: https://github.com/GoogleContainerTools/skaffold
skaffold-image-leeroy-web     image        url: localhost:5000/leeroy-web
target-image-6c8w9            image        url: gcr.io/dogfooding/tkn:latest
target-image-799wn            image        url: gcr.io/dogfooding/tkn:latest
```

# Steps to Reproduce the Problem

1.
2.
3.

# Additional Info

### PR DESCRIPTION
**Pull request by afrittoli**
Close #383

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Regenerate the manpages and docs with `make docs` and `make man` if needed.
- [ ] Run the code checkers with `make check`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

File: pkg/cmd/pipelineresource/list.go [hunk 1]
```diff
@@ -160,6 +160,8 @@ func details(pre v1alpha1.PipelineResource) string {
 	var key = "url"
 	if pre.Spec.Type == v1alpha1.PipelineResourceTypeStorage {
 		key = "location"
+	} else if pre.Spec.Type == v1alpha1.PipelineResourceTypeCloudEvent {
+		key = "targeturi"
```
Comment (sthaha) [inline pkg/cmd/pipelineresource/list.go]: isn't the key case sensitive? `targetURI?` ?

Comment (afrittoli) [reply pkg/cmd/pipelineresource/list.go]: A few lines above the key is set to lower case before it's checked, so `targetURI` would not match.

Comment (sthaha) [reply pkg/cmd/pipelineresource/list.go]: thank you!

**Pull request merged**